src/main/role.ts

```diff
@@ -0,0 +1,64 @@
+import { RowBasecampMappingMissingError } from "./error/rowBasecampMappingMissingError";
+import { getRowBasecampMapping } from "./row";
+
+export enum RoleStatus {
+    OBSOLETE,
+    SURVIVING,
+    NEW
```

Comment:
I was a little confused at first about what it meant for a role to be obsolete or surviving. Would something like `Removed` and `Unchanged or Existing` be more descriptive?

Reply:
I updated the enum to use EXISTING instead of SURVIVING and REMOVED instead of OBSOLETE